src/api/app/views/webui2/webui/project/_pulse_list.html.haml

```diff
@@ -0,0 +1,29 @@
+.col
+  .row
+    .col
+      .progress{ style: "height: 8px"}
```

Comment:
Please don't use static size units. They don't play well with responsive views. Also we should stick with one size unit. That would be `rem` in our case.